Userland/Applications/HexEditor/ValueInspectorModel.h

```diff
@@ -127,6 +130,12 @@ class ValueInspectorModel final : public GUI::Model {
             case UnsignedLong:
             case Double:
                 return 8;
+            case UTF8: {
+                auto utf8_view = Utf8View(m_values.at(index.row()));
+                if (utf8_view.validate())
+                    return static_cast<i32>(utf8_view.byte_length());
```

Comment:
This says multiple though.

Reply:
No, this just makes sure that all bytes that make up the UTF-8 code point are highlighted. We technically don't check the amount of UTF-8 code points here, but we never insert more than one in the first place.

Reply:
For example, if the code point starting at the cursor spans two bytes, it will actually highlight those two points when you click the UTF-8 row in the value inspector.